apps/web/app/(use-page-wrapper)/(main-nav)/availability/server-page.tsx

```diff
@@ -0,0 +1,49 @@
+import { _generateMetadata, getTranslate } from "app/_utils";
```

Comment:
`server-page.tsx` files are just copies of the original `page.tsx` so we can go back to server-side fetching more easily in the future

you can skip reviewing these files since they aren't consumed anywhere atm